tests/alive-tv/memory/memcmp-split.src.ll

```diff
@@ -0,0 +1,17 @@
+target datalayout = "e-p:64:64:64"
+; TEST-ARGS: -disable-undef-input
```

Comment:
I guess this is because the split increases # uses? If that's the case, please leave a comment

Reply:
The main reason (other test files as well) was the timeout. I couldn't make it raise timeout unless -disable-undef-input was given.
I think the tests still make sense without the existence of undef. I'll leave comments about these.